### PR TITLE
fabric difference between connecting to 4.0 and 4.1

### PR DESCRIPTION
**Pull request by recrwplay**

### CODE REVIEW

File: asciidoc/cypher-workflow.adoc [hunk 1]
```diff
@@ -274,23 +274,47 @@ include::{python-examples}/test_read_write_transaction_example.py[tags=read-writ
...
+
+// creates an index in bar
+session.run("USE bar CREATE INDEX bar_idx FOR (n:Person) ON n.id").consume()
+
+// targets System database
+session.run("SHOW USERS")
+----
+
+[NOTE]
+If you are connecting to *Neo4j 4.0*, some administrative operations must be carried out against the `system` database.
```
Comment (tobias-johansson) [inline asciidoc/cypher-workflow.adoc:315]: It's more than that. The `USE` keyword is also only available in Neo4j 4.1, so the whole example is 4.1-specific

Comment (tobias-johansson) [inline asciidoc/cypher-workflow.adoc:315]: It's more than that. The USE keyword is also only available in Neo4j 4.1, so the whole example is 4.1-specific

File: asciidoc/cypher-workflow.adoc [hunk 2]
```diff
@@ -308,6 +332,33 @@ Changing to any other database than the default database in Neo4j Community Edit
 
 Please note that the database that is requested must exist.
 
+The following example illustrates the concept of DBMS transactions, and shows how queries to multiple databases can be issued in one driver transaction. It is annotated with comments describing how each operation impacts database transactions.
+
+[source]
```
Comment (tobias-johansson) [inline asciidoc/cypher-workflow.adoc:337]: This example is also specific to Neo4j 4.1

**Review by tobias-johansson — COMMENTED** [review]
I'm thinking maybe we shouldn't update the 4.0 Driver manual with this?

Using these new capabilities in Neo4j 4.1 from a 4.0 driver is a bit limited anyway, since server-side routing is disabled in such situations (which may lead to confusing errors)

Perhaps we should (once the 4.1 drivers have been released) simply add a Note in the 4.0 driver manual telling the user that if they want to fully utilize the multi-db transactions and query routing in Neo4j 4.1 they should upgrade their driver to 4.1?

**Comment by recrwplay** [comment]
>Perhaps we should (once the 4.1 drivers have been released) simply add a Note in the 4.0 driver manual telling the user that if they want to fully utilize the multi-db transactions and query routing in Neo4j 4.1 they should upgrade their driver to 4.1?

Sounds like a good solution - should we also include the code examples in the 4.1 drivers doc at that point?

**Comment by davidoliverSP2** [comment]
@recrwplay / @tobias-johansson 

Is there any further action to this PR?

**Comment by davidoliverSP2** [comment]
Assuming this is no longer in scope.

**Pull request closed without merging**